ca_gazebo/scripts/publishtagspose.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python  
```

Comment:
rename the file to `publish_tags_poses.py `.

Reply:
Adressed